From 954100c3ff4e7cef4cb100cf05a306d65b680727 Mon Sep 17 00:00:00 2001
From: Mana Fukumoto <mana66354@gmail.com>
Date: Sat, 5 Oct 2024 22:48:42 +0900
Subject: [PATCH 3/3] =?UTF-8?q?nginx=E7=B7=A8=E9=9B=86?=
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

---
 docker/nginx/default.prod.conf | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/docker/nginx/default.prod.conf b/docker/nginx/default.prod.conf
index d32d42d..0cc8a75 100644
--- a/docker/nginx/default.prod.conf
+++ b/docker/nginx/default.prod.conf
@@ -24,7 +24,7 @@ server {
 
     location ~ \.php$ {
         fastcgi_split_path_info ^(.+\.php)(/.+)$;
-        fastcgi_pass unix:/var/run/php/php8.1-fpm.sock;
+        fastcgi_pass   php:9000;
         fastcgi_index index.php;
         include fastcgi_params;
         fastcgi_param SCRIPT_FILENAME $document_root$fastcgi_script_name;
-- 
2.46.2

